tests/Operation/ExplainTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace MongoDB\Tests\Operation;
+
+use MongoDB\Operation\Count;
+use MongoDB\Operation\Distinct;
+use MongoDB\Operation\Explain;
+
+class ExplainTest extends TestCase
+{
+    /**
+     * @requires PHPUnit 5.4.0
+     * @expectedException MongoDB\Exception\InvalidArgumentException
+     * @dataProvider provideInvalidConstructorOptions
+     */
+    public function testConstructorOptionTypeChecks(array $options)
+    {
+        $explainable = $this->createMock('MongoDB\Operation\Explainable');
```

Comment:
I didn't realize that `createMock()` was only added in later PHPUnit versions. You should be able to replace this with `$this->getMockBuilder('MongoDB\Operation\Explainable')->getMock();` for compatibility across all versions. That would allow this to be run on our PHP 5.x environments.